libs/blocks/tabs/tabs.css

```diff
@@ -492,6 +498,77 @@
   text-shadow: revert;
 }
 
+/* Segmented Control Variant */
+.tabs.segmented-control .tabList {
+  background: none;
+  box-shadow: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  border-radius: var(--m-rounded-corners);
+  overflow: hidden;
+}
+
+.tabs.segmented-control .tabList .tab-list-container {
+  background: #f8f8f8;
+  overflow: hidden;
+  border-radius: var(--m-rounded-corners);
+  padding: 1px;
+  min-width: 298px;
+}
+
+.tabs.segmented-control .tabList button {
+  padding: 10px;
+  border-radius: var(--m-rounded-corners);
+  border: none;
+  font-weight: 700;
+  color: var(--tabs-pill-bg-color);
+  font-size: 14px;
+  flex: 1 0 auto;
+}
+
+.tabs.segmented-control .tabList button[aria-selected="true"],
+.tabs.segmented-control .tabList button[aria-checked="true"] {
+  background: var(--tabs-pill-bg-color);
+  color: var(--color-white);
+}
+
+.tabs.segmented-control .tabList button.paddle {
+  position: absolute;
+  border-bottom: none;
+  width: 54px;
+  height: 40px;
+  box-shadow: none;
+  flex-shrink: 0;
+  justify-content: end;
+  padding: 0 !important;
```

Comment:
Specificity for this selector is already higher than the potentially conflicting `.tabs.segmented-control .tabList button`, so `!important` is not needed.